interfaces/cython/cantera/reactor.pyx

```diff
@@ -651,23 +651,23 @@ cdef class ExtensibleReactor(Reactor):
         def __set__(self, n):
             self.accessor.setNEq(n)
 
-    property vdot:
+    property expansion_rate:
```

Comment:
I think we need to keep the existing names in deprecated form, since `ExtensibleReactor` was part of the 2.6.0 release and these properties are meant for use by end users. (Only for the Python interface -- the C++ `ReactorDelegator` is effectively internal)

Reply:
Ah, sorry. For some reason I thought it was a new feature. Will use the usual deprecation sequence, and also update the new property to the updated syntax.